src/melee.cpp

```diff
@@ -990,6 +991,15 @@ void Character::reach_attack( const tripoint &p, int forced_movecost )
             !x_in_y( ( target_size * target_size + 1 ) * skill,
                      ( inter->get_size() * inter->get_size() + 1 ) * 10 ) ) {
             // Even if we miss here, low roll means weapon is pushed away or something like that
+            if( inter->has_effect( effect_pet ) || ( inter->is_npc() &&
+                    inter->as_npc()->is_friendly( get_player_character() ) ) ) {
+                if( query_yn( _( "Your attack will cause accidental injury, continue?" ) ) ) {
```

Comment:
```suggestion
                if( query_yn( _( "Your attack may cause accidental injury, continue?" ) ) ) {
```
I think it's not guaranteed to cause accidental injury.